src/coordination/include/coordination/register_main_replica_coordinator_status.hpp

```diff
@@ -59,5 +59,12 @@ enum class SetInstanceToMainCoordinatorStatus : uint8_t {
   FAILED_TO_CLOSE_LOCK
 };
 
+enum class AddCoordinatorInstanceStatus : uint8_t {
+  SUCCESS,
+  ID_ALREADY_EXISTS,
+  BOLT_ENDPOINT_ALREADY_EXISTS,
+  RAFT_ENDPOINT_ALREADY_EXISTS
```

Comment:
Isn't this the coordinator endpoint? I think we changed it from RAFT to coordinator

Reply:
yes